application/helpers/SurveyRuntimeHelper.php

```diff
@@ -910,7 +910,7 @@ private function checkClearCancel()
      */
     private function setPrevStep()
     {
-        if (isset($this->sMove)) {
+        if (isset($this->sMove) && $this->sMove != "") {
```

Comment:
`!empty($this->sMove)` ?

or `isset($this->sMove) && $this->sMove` since we use !=